WarmUp1/posNeg/src/index.ts

```diff
@@ -0,0 +1,74 @@
+
+/*
+Given 2 int values, return true if one is negative and one is positive. 
+Except if the parameter "negative" is true, then return true only if both are negative.
+
+
+posNeg(1, -1, false) → true
+posNeg(-1, 1, false) → true
+posNeg(-4, -5, true) → true
+
+*/
+
+export function posNeg(firstParameter: number, secondParameter: number, negParamter: Boolean): boolean { 
+
+    switch (negParamter) {
+
+        case false: {
+        
+            if ((firstParameter > 0 && secondParameter < 0)
+                || (firstParameter < 0 && secondParameter > 0)) {
+                return true;
+            }
+            
+            else {
+                return false;
+            }
+        }
+        
+        
+        case true: {
+            
+            if ((firstParameter < 0) && (secondParameter < 0)) {
+                return true;
+            }
+
+            else {
+                return false;
+            }
+        }
+            
+        default: {
+            return false
+        }
+    }
+}
+
+
+
+
+
+
+
+
+
+/*
```

Comment:
Please remove commented out code, we don't really need this here but it looks like this commented code is neater than the current solution, maybe a challenge would be to do this way?